// [[Reason: checker.typeToTypeNode deliberately fails on types that originate from node_modules.]] ////

//// [tests/cases/compiler/declarationEmitUsingTypeAlias1.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,5 +1,11 @@
 
+
+//// [src/index.d.ts]
+import { SomeType } from "some-dep";
+export declare const foo: (thing: SomeType) => SomeType;
+export declare const bar: (thing: SomeType) => Other;
+//# sourceMappingURL=index.d.ts.map
 /// [Errors] ////
 
 src/index.ts(7,39): error TS2304: Cannot find name 'Other'.
 src/index.ts(7,39): error TS4025: Exported variable 'bar' has or is using private name 'Other'.
